pkg/driver/controller.go

```diff
@@ -28,6 +28,21 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var (
+	// EBS volume capability. It could only be read/write on a single node at any given time.
+	volumeCaps = []csi.VolumeCapability_AccessMode{
+		{
+			Mode: csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
+		},
+	}
+
+	// controller capability
+	controllerCaps = []csi.ControllerServiceCapability_RPC_Type{
```

Comment:
I'm not sure if making these fields global is a better practice. It'd make it difficult to create a driver with custom capabilities for testing, for example.

Another option would be to create functions to set up and return the caps, and use these function in `New*Driver()`.

If we want to keep it like this, we may want to name these vars like `defaultVolume...` and use these names in the comments (e.g., `defaultVolumeCapabilities represents the EBS volume...`.

Reply:
> I'm not sure if making these fields global is a better practice. It'd make it difficult to create a driver with custom capabilities for testing, for example.

I feel we will never create a CSI EBS driver with a different capability eg, volume capability of `MULTI_NODE_MULTI_WRITER` since that is limited by EBS; or we will create a driver whose controller service capability is configurable, eg `CREATE_DELETE_VOLUME` is configurable since we always want dynamic provisioning. 

I kept variable name but updated the comments of those variable though.

Reply:
I don't feel like moving those vars to a global scope would make the code better, even though they might not be ever changed.

I agree that initializing them in `NewDriver` feels messy, but I'd rather create functions that return the default values and use those functions in `NewDriver` (if I'm not mistaken GCP driver does something like that).

However, I don't want to block this PR from being merged, so feel free to merge it if you think it's OK as is.